### DISCUSSION TITLE
Post Authentication Action after user creation

### DISCUSSION THREAD

**Discussion by danielmichalichyn** [zitadel/zitadel#6745] [Q&A]
When a user is created and the post authentication action is executed, the user object doesn't have the userId created. This is intentional? Can we have the user already commited/created into the database and the id of the user available for use in the action when this happens?

**Comment by hifabienne** [comment]
@livio-a @adlerhurst can you have a look at this?

**Reply by danielmichalichyn** [reply]
@livio-a  or @adlerhurst , do you guys have any update on this?

**Reply by adlerhurst** [reply]
sound like a bug to me. the id should be available at this point

**Reply by danielmichalichyn** [reply]
i'm creating a issue based on this discussion
